langtest/langtest.py

```diff
@@ -738,3 +738,31 @@ def import_edited_testcases(self, input_path: str, **kwargs):
         self._testcases.extend(temp_testcases)
 
         return self
+
+    def available_tests(test_type=None):
+        """
+        Returns a dictionary of available tests categorized by test type.
+
+        Args:
+            test_type (str, optional): The specific test type to retrieve. Defaults to None.
+
+        Returns:
+            tuple or dict: If test_type is provided, returns a dictionary containing available tests for that type.
+                        If test_type is None, returns a tuple with the message and the entire available_tests dictionary.
```

Comment:
Not sure to get why we are returning a dict in one case and a tuple in the other.. Maybe I'm missing something here but why don't we just return a prettify string here? Are we expecting the user to manipulate the returned value?

If we do expect the user to manipulate the returned value then I would return a dict with key test_type and value the list of available tests